app/models/Leaderboard.py

```diff
@@ -0,0 +1,32 @@
+from app.models.Answer import Answer
+from config import answers
+
+class Leaderboard(object):
+    """
+    Manages leaderboard for a post
+    """
+
+    def __init__(self, post_id, limit=answers['leaderboard_items']):
+        self.post_id = post_id
+        self.default_limit = limit
+
+    def get_answers(self, limit=None):
+        if limit is None:
+            limit = self.default_limit
```

Comment:
Rather than limiting the number of items in the leaderboard, why not make it scrollable past a certain length?

Reply:
It is but those are not loaded by default to avoid needing to have a giant leaderboard unless user clicks see more